### PR TITLE
unit test for addExistingBridgeToken

### PR DESCRIPTION
**Pull request by juniuszhou**
The pr add the unit test for addExistingBridgeToken in smart contract

### CODE REVIEW

File: smart-contracts/test/testBridgeBank.js [hunk 1]
```diff
@@ -632,5 +632,45 @@ describe("Test Bridge Bank", function () {
...
+      await expect(
+        state.bridgeBank
+          .connect(owner)
+          .addExistingBridgeToken(state.token1.address)
+      ).to.be.fulfilled;
+
+      inWhiteList = await state.bridgeBank.getCosmosTokenInWhiteList(state.token1.address);
+      expect(inWhiteList).to.be.equal(true);
+
+      inWhiteList = await state.bridgeBank.getCosmosTokenInWhiteList(state.token2.address);
```
Comment (smartyalgo) [inline smart-contracts/test/testBridgeBank.js:656]: Why do we need to repeat this test with token2?

Comment (juniuszhou) [reply smart-contracts/test/testBridgeBank.js:656]: just guarantee the token2 not in the whitelist before call batchAddExistingBridgeTokens.

**Review by smartyalgo — APPROVED** [review]

**Comment by smartyalgo** [comment]
Tests is failing for this one.

**Comment by juniuszhou** [comment]
> Tests is failing for this one.

fixed.

**Pull request merged**